### ISSUE TITLE
Create  initial Wikidata subset as KGTK files to support all downstream services

### ISSUE THREAD

**Issue by szeke** [usc-isi-i2/kgtk#207]
Done in https://github.com/usc-isi-i2/kgtk/blob/dev/examples/Wikidata%20Subsets.ipynb

Data published to the KGTK drive in datasets/wikidataos-v1

**Comment by dgarijo** [comment]
Why not in the public dataset drive? Is this not supposed to go in https://usc-isi-i2.github.io/wikidata-kgtk-releases/website/index.html?

**Comment by szeke** [comment]
Yes, will go there. The current files are preliminary as they are missing the edges for the properties. Once we debug it we will put it in the public site.
